src/common/components/KeypomLogo/KeypomLogo.tsx

```diff
@@ -2,7 +2,7 @@ import { HStack, Box, Text } from '@chakra-ui/react';
 
 export const KeypomLogo = () => {
   return (
-    <HStack spacing={{ base: 2, md: '2.5' }}>
+    <HStack as="a" href="/" spacing={{ base: 2, md: '2.5' }}>
```

Comment:
Do you think it’ll be better if we wrap this and use next/link instead?

Reply:
yeah, i think Link is better than `a`. Let me change it